sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -373,13 +374,12 @@ public class ChatClient {
         }
     }
 
-    /// Start push notifications. Receiving of notifications can be expected ~1 second after successfully registering.
+    /// Start push notifications. Receiving of notifications can be expected after successfully registering.
     /// - Parameters:
     ///   - deviceToken: APNS push token.
     ///   - completionHandler: Success indicates request to register for notifications has been received.
     public func startPushNotifications(
```

Comment:
Just wondering is it better to have a separate API called renewRegistration()? So instead of the whole process being invoked by client every 15 minutes, we just generate/rotate the keys. Or could we have a flag recording is it the first time start push notification, and avoid duplicate creation of objests?

Reply:
Yea on Android side we have renewRegistration(), but this "renew feature" is not within the scope in this release for IOS. For the second part of the comments, we have a flag "pushNotificationsStarted" inside PushNotificationClient to indicate if the PN has started. And thanks for sharing thoughts on how to generate object in a more efficient way when doing auto-renew - I will mark this and discuss further when we consider implementing this feature in next iterations.